### ISSUE TITLE
file repository: cannot assign an image to context site

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2268]
I am not sure if this is intentional. I imported a bunch (rather, a ton) of images and am giving some of them contexts. I appear unable to assign an image to a context that is a site, and I had expected to be able to do that.
![image](https://github.com/arch-kiosk/arch-kiosk-office/assets/49291909/8c75c710-197f-4f75-9c1b-4ed92a2a109c)

**Comment by lbestock** [comment]
Ah there already is a site map for that which might be the thing?

**Comment by lbestock** [comment]
No, it still fails for a site that has no site map, so that I would call a bug. Same error message.

**Comment by urapadmin** [comment]
Ja, site is a problem. You also cannot run a file import that imports a file for a site identifier. That is the same issue. If we allow it here we would allow it for the file import as well. But we can't allow it there because the site image is only one per site. The file import or rather the part that assigns the image to the identifier can't deal with that, yet.

**Comment by urapadmin** [comment]
see also #616

**Comment by urapadmin** [comment]
this is strangely connected to #2252

**Comment by urapadmin** [comment]
this is hard and now postponed. #2278 is now the workaround as long as we don't have a solution here.

**Comment by luizaogs** [comment]
I know how to get around it. Was just giving you an example of a scenario where having more site images not attached to site notes would be useful. I guess that's more tied to #616.

**Comment by urapadmin** [comment]
> I know how to get around it. Was just giving you an example of a scenario where having more site images not attached to site notes would be useful. I guess that's more tied to #616.
> 
> I can also open a new ticket, if that is better.

yes, that sounds indeed as if it is #616 but in the end all these belong together. We can leave it here.
